apps/flame-defi/app/config/index.ts

```diff
@@ -34,6 +34,18 @@ export interface AppConfig {
   feedbackFormURL: string | null;
   // List of networks to display in the network selector.
   networksList: FlameNetwork[];
+  // The default approval amount for tokens.
+  TokenDefaultApprovalAmount: string;
```

Comment:
This and the other new property can start lowercase. I typoed in my comment. I'll go ahead and approve, but fix this up first if you can!